LEGO1/realtime/vector.h

```diff
@@ -416,6 +408,11 @@ class Vector4 : public Vector3 {
 		m_data[3] = p_value;
 	} // vtable+0x84
 
+	float& operator[](int idx) { return m_data[idx]; }
```

Comment:
This function is being used, but not in any BETA10 functions we have matched so far. I'll keep looking.